lib/tracker.coffee

```diff
@@ -0,0 +1,48 @@
+# contants
+analyticsWriteKey = 'pDV1EgxAbco4gjPXpJzuOeDyYgtkrmmG'
+
+# imports
+_ = require 'underscore-plus'
+
+module.exports =
+class Tracker
+
+  constructor: (@analyticsUserIdConfig) ->
+    {allowUnsafeEval} = require 'loophole'
+    @pkg ?= require("../package.json")
+
+    # Setup Analytics
+    Analytics = null
+    allowUnsafeEval -> Analytics = require 'analytics-node'
+    @analytics = new Analytics analyticsWriteKey, flushAt: 1
+
+    # set a unique identifier
+    if not atom.config.get @analyticsUserIdConfig
+      uuid = require 'node-uuid'
+      atom.config.set @analyticsUserIdConfig, uuid.v4()
+    # identify the user
+    atom.config.observe @analyticsUserIdConfig, {}, (userId) =>
+      @analytics.identify {
+        userId: userId
+      }
+
+  track: (message) ->
+    message = event: message if _.isString(message)
```

Comment:
IMHO it makes the code harder to read, and the underscore makes exactly the same. Also it's bundled with atom, so I don't think it'll make any real difference if we drop it